example/gardener-local/kind/cluster/templates/_extra_port_mappings.tpl

```diff
@@ -29,6 +29,13 @@
 {{- end -}}
 {{- end -}}
 
+{{- define "extraPortMappings.gardener.operator.nginx" -}}
+{{- if and .Values.gardener.garden.deployed .Values.gardener.nginxIngress.deployed -}}
+- containerPort: 30448
+  hostPort: 8448
+{{- end -}}
+{{- end -}}
+
 {{- define "extraPortMappings.gardener.seed.nginx" -}}
```

Comment:
What about harmonizing this into a single `define` to not repeat the same content?